op-challenger/metrics/metrics.go

```diff
@@ -58,6 +62,12 @@ func NewMetrics() *Metrics {
 			Name:      "up",
 			Help:      "1 if the op-challenger has finished starting up",
 		}),
+		cannonExecutionTime: factory.NewHistogram(prometheus.HistogramOpts{
+			Namespace: Namespace,
+			Name:      "cannon_execution_time",
+			Help:      "Time (in seconds) to execute cannon",
+			Buckets:   append([]float64{1.0, 10.0}, prometheus.ExponentialBuckets(30.0, 2.0, 14)...),
```

Comment:
What about just using:
```suggestion
			Buckets:   append([]float64{1.0, 10.0}, prometheus.ExponentialBucketsRange(30.0, 60.0 * 3.0, 14)...),
```

Then we're defining the range we want and the number of buckets and letting prometheus do the math. :)

Reply:
That looks good to me - can always lower or raise the max when we start testing metrics :)

Reply:
Yeah 3 hours is pretty arbitrary but I think we'd want to find a faster CPU if cannon is taking longer than that...